rust/src/operations/filesystem_check.rs

```diff
@@ -154,10 +147,11 @@ impl FileSystemCheckPlan {
         }
 
         commit(
-            store,
-            version + 1,
+            self.store.as_ref(),
             &actions,
             DeltaOperation::FileSystemCheck {},
+            snapshot,
+            // TODO pass through metadata
```

Comment:
I plan to do a pass through all our operations in the next Pr, where also I'll be focussing on testing the `commit` function, and addressing these kinds of TODOs.